Include/internal/pycore_magic_number.h

```diff
@@ -0,0 +1,288 @@
+#ifndef Py_INTERNAL_MAGIC_NUMBER_H
+#define Py_INTERNAL_MAGIC_NUMBER_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef Py_BUILD_CORE
+#  error "this header requires Py_BUILD_CORE define"
+#endif
+
+/*
+
+Magic word to reject .pyc files generated by other Python versions.
+It should change for each incompatible change to the bytecode.
+
+The value of CR and LF is incorporated so if you ever read or write
+a .pyc file in text mode the magic number will be wrong; also, the
+Apple MPW compiler swaps their values, botching string constants.
+
+There were a variety of old schemes for setting the magic number.
+The current working scheme is to increment the previous value by
+10.
```

Comment:
This looks like it needs updating? It seems from Python 3.5.2 / 3351 that numbers were no longer incremented by 10.